src/gather/api.py

```diff
@@ -49,6 +50,23 @@ class Collector(object):
     and *collect*-ing them when they need to be used.
     """
 
+    @staticmethod
+    def one_of(_registry, _effective_name, objct):
```

Comment:
I wanted to keep the module exports to a reasonable amount, plus this function does not make a lot of sense on its own, but only as a strategy. In fact, the doc string will cover that bit :)